### ISSUE TITLE
Use of LOCALIZATION_PREFIXES causes datamodel labels to become immune to translation hot reload

### ISSUE THREAD

**Issue by mkahvi** [foundryvtt/foundryvtt#11762]
### What happened?

I used LOCALIZATION_PREFIXES on my datamodel, rendered some sheets, changed some strings in my en.json with hot reload enabled, saw no change.

This feels like a bug, but I can see it being intentional.

Considering the keys aren't even stored in the datamodel to begin with, it seems quite feasible to re-translate them on demand.

### What ways of accessing Foundry can you encounter this issue in?

- [ ] Native App (Electron)
- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other

### Reproduction Steps

Define datamodel with its labels and hints provided by the datamodel localization process.

### What core version are you reporting this for?

12.331

### Bug Checklist

- [ ] The issue occurs while all Modules are disabled

**Comment by aaclayton** [comment]
Yeah, I think on a hot reload we do not repeat data model localization.

**Comment by esheyw** [comment]
This seems like just removing the `game.i18n.localize` calls in `Localization.#localizeSchema` would let the labels and such be localized at application render like normal, no?

**Comment by aaclayton** [comment]
The entire point of this is to *pre-localize* these ordinarily static labels and hints to improve efficiency at render time.

**Comment by JPMeehan** [comment]
Is it possible there might be some kind of core dev-mode that can be tracked so people know whether to defer localization? You see similar issues with like `CONFIG.DND5E` getting pre-localized, which makes perfect sense for users but breaks hot reload in dev.

**Comment by mkahvi** [comment]
I don't believe there is such functionality. Core Foundry is doing double localization in bunch of places still. I made a module for detecting those to avoid translations getting mangled in my own projects, required just patching the localization functions to make them check if translation exists.

**Comment by JPMeehan** [comment]
Avoiding double localize calls seems like a broader project that needs to happen in light of the collision in French between `Spell` and `Sort`

**Comment by aaclayton** [comment]
The outcome of our discussion within the team is that this is `wontfix` because - by design - prelocalization only happens once on initial load and the complexity involved in making it a repeatable process is not worth the development effort it would require.
